powerquery-docs/connectors/bloomberg-data-and-analytics.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Power Query Bloomberg Data and Analytics connector
 description: Provides basic information and prerequisites for the Bloomberg Data and Analytics connector, descriptions of the optional input parameters, and discusses limitations and issues you might encounter.
-author: kanstantsin-chernik
+author: Shrewbitz
```

Comment:
```suggestion
author: DougKlopfenstein
```

Reply:
Does author mean something different than the person who wrote the documentation in this instance?

Reply:
It usually matches the ms.author, but it doesn't have to. We can keep it as-is.